mindsdb/integrations/handlers/utilities/query_utilities/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .select_query_utilities import SELECTQueryParser, SELECTQueryExecutor
-from .insert_query_utilities import INSERTQueryParser
 from .update_query_utilities import UPDATEQueryParser, UPDATEQueryExecutor
-from .delete_query_utilities import DELETEQueryParser, DELETEQueryExecutor
+#from .delete_query_utilities import DELETEQueryParser, DELETEQueryExecutor
```

Comment:
Why have these imports been commented out?

Reply:
Because they currently do not exist, but if merged would make it slightly easier for other devs on DELETE, INSERT if I don't get to it first.

Reply:
@Sohlar Please check the staging branch and sync it with yours. They do exist.

Reply:
The imports of the `delete_query_utilities` are still commented out.